cluster-autoscaler/cloudprovider/mcm/mcm_cloud_provider_test.go

```diff
@@ -51,13 +54,14 @@ type setup struct {
 	machines                          []*v1alpha1.Machine
 	machineSets                       []*v1alpha1.MachineSet
 	machineDeployments                []*v1alpha1.MachineDeployment
+	deployments                       []*v1.Deployment
```

Comment:
We can change this to `mcmDeployment` instead of `deployments` and the type to `*v1.Deployment`. We only check the MCM deployment, so let's keep the test specific to that.